### PR TITLE
Theme

### PR DESCRIPTION
**Pull request by malin-mallan**
Added a colortheme and numbers

We could not add a specific darktheme

### CODE REVIEW

File: src/components/RootNavigator.tsx [hunk 1]
```diff
@@ -12,12 +12,14 @@ const Tab = createMaterialBottomTabNavigator();
...
+
   return (
-    <Tab.Navigator initialRouteName='Activity' shifting={true} sceneAnimationEnabled={false} labeled={false}>
-      <Tab.Screen name='Activites' component={ActivityScreen} options={{ tabBarIcon: 'plus', title: lang.navigationLabelActivites }} />
+    <Tab.Navigator initialRouteName='Activity' shifting={true} sceneAnimationEnabled={false} labeled={true} barStyle={{backgroundColor: colors.primary}} >
+      <Tab.Screen name='Activites' component={ActivityScreen} options={{ tabBarIcon: 'plus', title: lang.navigationLabelActivites  }} />
       <Tab.Screen name='Values' component={ValuesScreen} options={{ tabBarIcon: 'heart', title: lang.navigationLabelValues }} />
       <Tab.Screen name='Calendar' component={CalendarScreen} options={{ tabBarIcon: 'calendar', title: lang.navigationLabelCalendar }} />
-      <Tab.Screen name='Information' component={InformationScreen} options={{ tabBarIcon: 'information', title: lang.navigationLabelInformation }} />
+      <Tab.Screen name='Information' component={InformationScreen} options={{ tabBarIcon: 'information', title: lang.navigationLabelInformation}} />
```
Comment (EMattfolk) [inline src/components/RootNavigator.tsx]: Something happened with the whitespace. I think it should be like in the unchanged rows.

File: src/components/activity/ActivityHistory.tsx [hunk 2]
```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import { View, FlatList, Pressable } from 'react-native';
-import { Surface, List, Caption, FAB, Title, useTheme } from 'react-native-paper';
+import { Surface, List, Caption, FAB, Title, useTheme} from 'react-native-paper';
```
Comment (EMattfolk) [inline src/components/activity/ActivityHistory.tsx]:
```suggestion
import { Surface, List, Caption, FAB, Title, useTheme } from 'react-native-paper';
```

File: src/components/values/CategoryButton.tsx [hunk 3]
```diff
@@ -14,6 +14,7 @@ export const CategoryButton = (props: CategoryButtonProps) => {
   const [values, modifyValues] = Storage.useValues();
   const [people, modifyPeople] = Storage.usePeople();
   const [showPortal, setShowPortal] = useState(false);
+  const { colors, roundness} = useTheme();
```
Comment (EMattfolk) [inline src/components/values/CategoryButton.tsx]:
```suggestion
  const { colors, roundness } = useTheme();
```

File: src/components/values/CircleButton.tsx [hunk 4]
```diff
@@ -1,5 +1,5 @@
 import React from 'react';
-import { Surface, IconButton } from 'react-native-paper';
+import { Surface, IconButton, useTheme} from 'react-native-paper';
```
Comment (EMattfolk) [inline src/components/values/CircleButton.tsx]:
```suggestion
import { Surface, IconButton, useTheme } from 'react-native-paper';
```

File: src/CustomTheme.tsx [hunk 5]
```diff
@@ -1,11 +1,14 @@
-import { DefaultTheme, DarkTheme } from 'react-native-paper';
+import { DefaultTheme, DarkTheme} from 'react-native-paper';
```
Comment (EMattfolk) [inline src/CustomTheme.tsx]:
```suggestion
import { DefaultTheme, DarkTheme } from 'react-native-paper';
```

**Review by EMattfolk — APPROVED** [review]
Looks good! Had some minor whitespace fixes.
Perhaps we should turn down the roundness a bit?

**Review by Neathan — APPROVED** [review]
I think the changes looks good overall!

The effects of "roundness" should perhaps be looked over since many of them seeme unintentional. Such as menus.

Another thing to point out is that when adding elevation the feature must be tested on both Android and iOS since it sometimes makes components non-interactable if they are elevated. I have not tested this on extensively for this code but it should be done before merge.

Other than that i think the theme itself is a clear improvement of what we had before. :)

**Pull request merged**